### ISSUE TITLE
pool

### ISSUE THREAD

**Issue by Krismaty505** [UnamSanctam/SilentCryptoMiner#1784]
which pool should I choose for monero?

**Comment by UnamSanctam** [comment]
Usually I recommend 2miners or nanopool, many other pools sometimes ban you.

**Comment by Krismaty505** [comment]
> Обычно я рекомендую 2miners или nanopool, многие другие пулы иногда забанивают вас.

Should I just insert it like that? :  xmr.2miners.com:2222

**Comment by UnamSanctam** [comment]
Yes that's correct, or `xmr.2miners.com:12222` for SSL (secure connection).

**Comment by UnamSanctam** [comment]
Can you run the checker and post a screenshot of it here?

**Comment by Krismaty505** [comment]
> Можете ли вы запустить программу проверки и опубликовать ее скриншот здесь?

Watchdog Running: Yes
Miners:
--algo=rx/0  --url=xmr.2miners.com:2222 --user="41vbU13PDu72DoH8Kg2AWke31AZhpfX4ZHx7c4AQz6NYitaPdbMraGtAcxeHDJMAGyP9hBbexfVWiJhQViLN9gNQ9FZbZRM" --pass="" --cpu-max-threads-hint=40 --cinit-winring="aoekoghvenxe.sys" --cinit-stealth-targets="Taskmgr.exe,ProcessHacker.exe,perfmon.exe,procexp.exe,procexp64.exe" --cinit-version="3.3.0" --tls --cinit-idle-wait=3 --cinit-idle-cpu=80 --cinit-id="wcssoegzyptuzolk"
GPUs:
 AMD Radeon(TM) RX Vega 10 Graphics
Compatible GPU found: True

**Comment by UnamSanctam** [comment]
Since you have SSL enabled (`--tls`) then you should use port 12222, so: `xmr.2miners.com:12222`

**Comment by Krismaty505** [comment]
> xmr.2miners.com:12222
What if I downloaded the miner and then deleted all the files(((((((

**Comment by Krismaty505** [comment]
> Поскольку у вас включен SSL (`--tls`), вам следует использовать порт 12222, чтобы: `xmr.2miners.com:12222`

What if I downloaded the miner and then deleted all the files

**Comment by UnamSanctam** [comment]
Depends on what you mean, do you mean that you started the miner and then removed the uninstaller so that you can't remove the installed miner?

**Comment by Krismaty505** [comment]
Well, which were after the creation in the builder

**Comment by Krismaty505** [comment]
> Если вам нужно удалить майнер вручную, то это самый простой метод:
> 
> * Откройте планировщик задач
> * Найдите задачу с "Именем записи", которое вы выбрали в конструкторе майнеров (`GoogleUpdateTaskMachineQC` по умолчанию)
> * Удалить задачу
> * Немедленно перезагрузите компьютер, если вы работаете слишком медленно, то сторожевой таймер повторно выполнит задачу

And which setting is responsible for the slow receipt of the hash?

**Comment by UnamSanctam** [comment]
Sorry could you rephrase that? Do you mean the time that it takes`for the hashrate to appear in the pool?

**Comment by Krismaty505** [comment]
Remind me what the SSL/TLS setup is responsible for

**Comment by Krismaty505** [comment]
> SSL означает, что соединение с пулом зашифровано, что означает, что оно не может быть прочитано или изменено какой-либо третьей стороной. И поскольку майнинг основан на удаче, то в зависимости от пула хешрейт должен повышаться и понижаться, тогда вы можете посмотреть на средний хешрейт за несколько часов (например, 6 часов), чтобы увидеть реальный хешрейт.

Ok. Thanks

**Comment by Krismaty505** [comment]
Hi. The graph on 2miner is not shown again, writes "There is no available data on the miner". What to do? I have already launched it, it works. I'll send a photo in the checker right now

**Comment by Krismaty505** [comment]
![Uploading IMG_20230902_134148_254.jpg…]()

**Comment by Krismaty505** [comment]
![Uploading Screenshot_20230902-153818.jpg…]()

**Comment by Krismaty505** [comment]
![Uploading Screenshot_20230902-153818.jpg…]()
